README.md

```diff
@@ -27,12 +27,42 @@ From this SQLDelight will generate a `Database` Kotlin class with an associated
 
 #### Android
 ```groovy
+android {
+  ...
+  sourceSets {
+    main.java.srcDir("src/main/sqldelight")
```

Comment:
this shouldnt be needed

Reply:
Without that, the sqldelight folder doesn't appear at all when using the "Android" project view

![image](https://user-images.githubusercontent.com/7407345/74571165-5db2fa80-4f75-11ea-8312-3e902f0f3dbd.png)

I know it's not ideal to put them in the java SourceSet, but I haven't found any better way to get the sqldelight source files accessible like this

![image](https://user-images.githubusercontent.com/7407345/74571270-7f13e680-4f75-11ea-9c47-e820b87f7a08.png)

Reply:
I did check and the contents of the sqldelight folder doesn't end up pulled into the APK or anything, so while it's slightly hacky, I think it's okay